diff_drive_node:
  ros__parameters:
    names: [ "W1_left", "W2_right" ]
    families: [ "DiffDrive" ]
